app/src/components/layout/LogoComponents/DefaultNavLogo.tsx

```diff
@@ -5,26 +5,36 @@ import { Link } from 'react-router-dom';
 import { pxToRem } from 'src/styled-theme';
 import { BlueLogo, ExpLogo } from '../../logos';
 
-export const DefaultNavLogo: React.FC = () => (
-  <NavLogoLink to="/">
-    <BlueLogo />
-    <ExplorerLogo />
-  </NavLogoLink>
-);
+export const DefaultNavLogo: React.FC = () => {
+  return (
+    <DefaultNavLogoContainer>
+      <NavLogoLink to="/">
+        <BlueLogo />
+        <ExplorerLogo />
+      </NavLogoLink>
+    </DefaultNavLogoContainer>
+  );
+};
 
-export const NavLogoLink = styled(Link)`
+const NavLogoLink = styled(Link)`
+  /* TODO:Remove these styles once new logo is added */
+  display: flex;
+  width: ${pxToRem(140)};
+`;
+
+export const DefaultNavLogoContainer = styled.div`
+  /* TODO:Remove these styles once new logo is added */
```

Comment:
Do we have a ticket number associated with this TODO? It might be good to explicitly state it in the comments here.

Reply:
There isn't unforturnately. The re-branding meeting was only a month ago, so seems like it could happen at anytime.  I will check in with Megan.

Reply:
Megan said it's not going to happen before the upcoming release, so removed them.

Reply:
OK sounds good. I think it would be a good habit for us to start writing the ticket number with the TODO comment (if we have one).